plasmapy/physics/parameters.py

```diff
@@ -210,10 +210,14 @@ def Alfven_speed(B, density, ion="p+", z_mean=None):
 @utils.check_relativistic
 @utils.check_quantity(
     T_i={'units': u.K, 'can_be_negative': False},
-    T_e={'units': u.K, 'can_be_negative': False}
+    T_e={'units': u.K, 'can_be_negative': False},
+    n_e={'units': u.m ** -3, 'can_be_negative': False},
+    k={'units': u.m ** -1, 'can_be_negative': False}
     )
 def ion_sound_speed(T_e,
                     T_i,
+                    n_e,
+                    k,
```

Comment:
I wonder whether it might not be possible to supply default arguments for these two, so that the default behavior is the non-dispersive limit we had previously?

Reply:
That would make a lot more sense! I'll work on updating it...

Reply:
That's a good question! It is getting late in my timezone, though, and I'm rather tired. Would you mind if I left the pondering of it and digging into the code for tomorrow? Alternatively, you could ask in [our chatroom](https://riot.im/app/#/room/#plasmapy:openastronomy.org) if anyone has any ideas!

Reply:
I wouldn't mind at all! Feel free to look through it whenever's best for you. :)